starfyre/compiler.py

```diff
@@ -56,10 +74,28 @@ def remove_empty_lines_from_end(lines):
 
     # regex pattern to match if a line is a css import, e.g. import "style.css"
     css_import_pattern = re.compile(r"^import\s[\"\'](.*?\.css)[\"\']")
-    
+
     with open(fyre_file_name, "r") as fyre_file:
         for line in fyre_file.readlines():
             css_import_match = css_import_pattern.search(line)
+
+            # check for fyre import styles
+            has_fyre_import = check_import_line(line=line, project_dir=project_dir)
+
+            # If the line is a fyre import statement, modify it to ensure proper resolution
+            if has_fyre_import:
+                # Split the line into module part and imported component
+                module_part, imported_component = line.split(" import ")
+
+                # Extract the file name to import from the module part
+                file_to_import = module_part.split(".")[-1]
+
+                # Get the name of the project directory
+                project_dir_name = str(project_dir).split("/")[-1]
+
+                # Modify the line to use the resolved import path
+                line = f'from {project_dir_name}.build.{file_to_import} import {imported_component}'
+
```

Comment:
These lines are used to resolve the import properly so that Starfyre can recognise the import. So because after compilation and transpilation our IR files go to the build folder, we need to fix how they are imported while being compiled. 
Before all files were just in on directory now because we have an inner directory the normal way of copying the exact same import statement won't work for **Absolute** imports. Relative imports are resolved automatically by Python